dotcom-rendering/src/components/Avatar.tsx

```diff
@@ -8,6 +8,7 @@ const picture = css`
 	height: 100%;
```

Comment:
Does this have the same result without needing to set min-width?

```suggestion
	height: auto;
```